libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1381,19 +1381,43 @@ namespace SohImGui {
 
             for (const auto& category : windowCategories) {
                 ImGui::SetCursorPosY(0.0f);
-                if (ImGui::BeginMenu(category.first.c_str())) {
-                    for (const std::string& name : category.second) {
-                        std::string varName(name);
-                        varName.erase(std::remove_if(varName.begin(), varName.end(), [](unsigned char x) { return std::isspace(x); }), varName.end());
-                        std::string toggleName = "g" + varName + "Enabled";
-
-                        EnhancementCheckbox(name.c_str(), toggleName.c_str());
-                        customWindows[name].enabled = CVar_GetS32(toggleName.c_str(), 0);
+                if (category.first != "Randomizer") {
```

Comment:
why does this need to be excluded like this? Is there a way to fit what we're doing into this pattern?

Reply:
The only way I could see is reverse the conditional and if it's met, add the new option just before or after this for loop `for (const std::string& name : category.second) {}`.

I did this originally (you can see it in the first commit of the PR), but it didn't feel great to me. If we ever want to change the order of the randomizer settings and item tracker or insert options inbetween them, that's also not possible if we would do it like that.

Another option is register the randomizer settings menu and the item tracker menu in an entirely different way so it doesn't show up in the autopopulate function here. Basically get rid of `SohImGui::AddWindow("Randomizer", "Item Tracker", DrawItemTracker);` and instead register it the same way the controller configuration menu does it.